### DISCUSSION TITLE
image entity

### DISCUSSION THREAD

**Discussion by Thomas55555** [Thomas55555/husqvarna_automower#503] [Ideas]
Home assistant introduces with 2023.7.0 a new image entity.
Looks for me more suitable than the camera entity. @prairiesnpr: What do you think?

**Comment by prairiesnpr** [comment]
Looking through the docs, it does seem like it's a better approach, we currently only recreate the image when it changes anyway, but the front end caching should make it load faster. It's derived from the camera entity, so should be a fairly easy move. I'll start to look at it, probably wait for a PR until we see 2023.7.1, assuming we want to break backwards compatiblity that fast. Mostly I just don't want to upgrade my production core instance, it's a huge pain to update.
